source/_drafts/Feature-Spotlight-Accessibility.md

```diff
@@ -0,0 +1,60 @@
+
+Supporting hearing impairment is not something that we can do for the users of Video.js. Instead we must indirectly support these users by supporting captions in their videos. In Video.js we have had support for captions/subtitles for some time, internally they are called `TextTracks`.
+
+Supporting vision impairment is harder, but fully in our control. To support this group of users our player must be accessible to "screen readers". A "Screen reader" is an application that reads elements off of the screen to the user (as the name implies). HTML has certain rules that must be followed so that a page can be accessible. We will go over the basics of these rules in the next section.
+
+Here are some popular screen readers that are actually used in the wild:
+
+* [VoiceOver for macOS](https://en.wikipedia.org/wiki/VoiceOver)
+* [JAWS for Windows](https://en.wikipedia.org/wiki/JAWS_(screen_reader))
+
+### How do you make a web application screen reader accessible?
+The most basic way to make a website accessible is to add the `role=""` attribute to an HTML element.
+
+> A list of roles can be found on [Mozilla Developer Network](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques#Composite_roles).
+
+Some Examples (assume all of these are plain `div`s):
+* A `MuteToggle` button would have a `role` attribute with a value of `button` like this: `role="button"`
+A `VolumeBar` slider would have a `role` attribute with a value of `slider` like this: `role="slider"`.
+
+If you use the native `<button>` element, you will not have to use the button role. This is the recommended way to make an element a button to the screen reader. In some cases (such as in Video.js) it will not be possible to use the native button element. So you will have to role instead.
+
+Another important point is that all ARIA controls must be usable with the keyboard alone. This is somewhat outside of the scope of this blog post but, usually you will have to add `tabindex`s to elements.
+
+After adding roles to the controls and content in your webpage. The next thing to look over are `aria` attributes. For instance we use `aria-live` for our `ProgressBar` slider. This is because the `ProgressBar` is always updating when a video is playing.
+
+> A list of attributes can be found in the [ARIA specification](https://www.w3.org/TR/wai-aria-1.1/)
+
+Finally you need to add an accessible name an element so that it can be referred to correctly. For instance a "Mute Button" with only a `role` of `button` would be read to the user as "button". If you add an accessible name to it, it will then be read as "Mute Button". In Video.js we refer to the accessible name as "Control Text". When we change the "Control Text" for a `Component`. A span with the specified text is added as a child element to the `Component` in question. This will let the screen reader know how to refer to the control.
```

Comment:
"add an accessible name *to* the element".

Also, can you elaborate on what "accessible name" means and how to add it?